### DISCUSSION TITLE
Is there a defined milestone for next(s) releases(s)?

### DISCUSSION THREAD

**Discussion by alejandrofm** [delta-io/delta-rs#1605] [General]
Hi! I saw the Gdoc with the main points for H12023, want to know if there are milestones/release schedules, mainly because I saw some issues fixed and want to know approximately when the next release will come out.
Thanks!!

**Comment by wjones127** [comment]
We don't have a fixed or predetermined release schedule at the moment.

But I'll work on getting a Python release out in the next week.

**Reply by alejandrofm** [reply]
Thank you @wjones127 !

**Comment by rtyler** [comment]
[this milestone](https://github.com/delta-io/delta-rs/milestone/2) is what @wjones127 is referring to.

We're experimenting more with Milestones to help communicate this sort of thing in the future.
